src/cli/link.js

```diff
@@ -18,24 +17,19 @@ export const builder = yargs => {
 };
 
 export async function handler(args) {
```

Comment:
There's a description string and yargs.options statement for every handler.
Those two should be documentation enough.